jmix-pivottable/pivottable-flowui-kit/src/main/resources/META-INF/frontend/src/pivot-table/jmix-pivot-table.js

```diff
@@ -106,6 +109,8 @@ export class JmixPivotTable extends ElementMixin(DisabledMixin(ThemableMixin(Pol
     _layout() {
         const container = document.createElement('div');
         container.className = 'pivot-table-output';
+        this.pivotTableId = JmixPivotTable.pivotTableCount++;
```

Comment:
You absolutely don't need this. Just store created layout in the instance field, e.g. `this._outputDiv`.